Remix/ZAPRemix/src/main/kotlin/ExtensionHunt.kt

```diff
@@ -0,0 +1,43 @@
+package org.zaproxy.zap.extension.hunt
+
+import org.parosproxy.paros.extension.ExtensionAdaptor
+import org.parosproxy.paros.extension.ExtensionHook
+import org.parosproxy.paros.network.HttpSender
+import java.awt.EventQueue
+
+class ExtensionHunt : ExtensionAdaptor(NAME) {
+    private val huntPanel = HuntPanel()
```

Comment:
The creation of the UI panel should be delayed until it's know that there's a view (e.g. `initView(ViewDelegate)` or in the hook check) to work properly in headless/daemon.

Reply:
Thanks for the review @thc202! I'll get that fixed ASAP!